### PR TITLE
Collection of minor fixes for Panel 1.5.1

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
- [x] fix fastapi example
- [x] replace iframe embed link in pycafe guide.
- [x] replace `--autoreload` with `--dev` in docs.
- [x] Add link to AnyWidget Front End Module (AFM) specification.

### CODE REVIEW

File: panel/command/serve.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ class Serve(_BkServe):
         )),
         ('--autoreload', dict(
             action  = 'store_true',
-            help    = "Whether to autoreload source when script changes."
+            help    = "Whether to autoreload source when script changes. We recommend using --dev instead."
```
Comment (hoxbro) [inline panel/command/serve.py:241]: This is correct for now, but there could be a future where `--dev` does more than `--autoreload`

Comment (MarcSkovMadsen) [reply panel/command/serve.py:241]: Is it just for info? Or should I remove the recommendation?

**Comment by MarcSkovMadsen** [comment]
This PR is growing. So I would like it reviewed and merged while its still possible to review.

**Review by philippjfr — APPROVED** [review]

**Comment by philippjfr** [comment]
Happy with this, thanks! I think recommending `--dev` over `--autoreload` is fine even if we expand the definition of `--dev` later.

**Pull request merged**